test/parallel/test-assert.js

```diff
@@ -726,6 +726,58 @@ assert.throws(
              'assert.ok(null)\n'
   }
 );
+assert.throws(
+  () => {
+    // This test case checks if `try` left brace without a line break
+    // before the assertion causes any wrong assertion message.
+    // Therefore, don't reformat the following code.
```

Comment:
```suggestion
    // Therefore, don't reformat the following code.
    // Refs: https://github.com/nodejs/node/issues/30872
```

Reply:
@cola119
Thanks for your suggestion. I will add the issue link.